From a39631b964423f6e24705fdb24a5c0403287d2a6 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Mon, 26 Apr 2021 17:57:14 +0300
Subject: [PATCH 1494/1921] arm64: dts: marvell: remove comphy nodes from
 pcie-ep

The new driver does not require any pcie comphy skip-config
settings, so remove them from pcie-ep dtsi file.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I25173386cd427925549570bea68d9698e9cacba2
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50634
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Ben Peled <Ben.Peled@cavium.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn913x-ep.dtsi | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn913x-ep.dtsi b/arch/arm64/boot/dts/marvell/cn913x-ep.dtsi
index 1f5cd2060e11..2b7bbdae2184 100644
--- a/arch/arm64/boot/dts/marvell/cn913x-ep.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn913x-ep.dtsi
@@ -63,22 +63,6 @@
 	status = "okay";
 };
 
-&cp0_comphy0 {
-	phy-skip-config;
-};
-
-&cp0_comphy1 {
-	phy-skip-config;
-};
-
-&cp0_comphy2 {
-	phy-skip-config;
-};
-
-&cp0_comphy3 {
-	phy-skip-config;
-};
-
 &cp0_armada_ep {
 	status = "okay";
 };
-- 
2.31.1

